variables.tf

```
@@ -53,7 +53,7 @@ variable "validation_method" {
 
 variable "validation_option" {
   description = "The domain name that you want ACM to use to send you validation emails. This domain name is the suffix of the email addresses that you want ACM to use."
-  type        = map(string)
+  type        = map(any)
```

ugh, these types are more pain that value currently. lets just go with `any` for now till we get to v1.3 support
```suggestion
  type        = any
```

---

@bryantbiggs updated vars and README. ( even map(map(string)) would have worked as I tested. but that looked to ugly to me *G)

---

ya, its a shame - but once we reach a point where we can set v1.3 as minimum we can have better defined variable data structures. Thanks for the fix!

---

I don't think we can/should consider upgrading to 1.3 earlier than at least 6-10 months after it will be released. For now, `any` is the most flexible way + we have the examples to see the structure.